models/iosxr/acls/iosxr_acls.yaml

```diff
@@ -0,0 +1,473 @@
+                        net_unreachable:
+                          description: Net unreachable
+                          type: bool
+                        network_unknown:
+                          description: Network unknown
+                          type: bool
+                        no_room_for_option:
+                          description: Parameter required but no room
+                          type: bool
+                        option_missing:
+                          description: Parameter required but not present
+                          type: bool
+                        packet_too_big:
+                          description: Fragmentation needed and DF set
+                          type: bool
+                        parameter_problem:
+                          description: All parameter problems
+                          type: bool
+                        port_unreachable:
+                          description: Port unreachable
+                          type: bool
+                        precedence_unreachable:
+                          description: Precedence cutoff
+                          type: bool
+                        protocol_unreachable:
+                          description: Protocol unreachable
+                          type: bool
+                        reassembly_timeout:
+                          description: Reassembly timeout
+                          type: bool
+                        redirect:
+                          description: All redirects
+                          type: bool
+                        router_advertisement:
+                          description: Router discovery advertisements
+                          type: bool
+                        router_solicitation:
+                          description: Router discovery solicitations
+                          type: bool
+                        source_quench:
+                          description: Source quenches
+                          type: bool
+                        source_route_failed:
+                          description: Source route failed
+                          type: bool
+                        time_exceeded:
+                          description: All time exceededs
+                          type: bool
+                        timestamp_reply:
+                          description: Timestamp replies
+                          type: bool
+                        timestamp_request:
+                          description: Timestamp requests
+                          type: bool
+                        traceroute:
+                          description: Traceroute
+                          type: bool
+                        ttl_exceeded:
+                          description: TTL exceeded
+                          type: bool
+                        unreachable:
+                          description: All unreachables
+                          type: bool
```

Comment:
Do we have icmpv6 and other v6 protocols for ipv6 access-list ? If yes and if they have sub options, they should be added under protocol_options

Reply:
Done. Argspec updated in https://github.com/ansible/ansible/pull/66207/files.